src/main/java/com/pmm/budgetcalculator/expensing/controller/ExpenseTypeController.java

```diff
@@ -0,0 +1,40 @@
+package com.pmm.budgetcalculator.expensing.controller;
+
+import com.pmm.budgetcalculator.expensing.entity.ExpenseType;
+import com.pmm.budgetcalculator.expensing.service.ExpenseTypeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("expenses/expense_types")
```

Comment:
We should design our API in way that every entity has it's own path in url. So, after /expenses we have all operations on expenses and on localhost:8080/expense_types we have all operations on expense types